fastlane/lib/fastlane/cli_tools_distributor.rb

```diff
@@ -47,6 +47,14 @@ def take_off
           Fastlane::CommandsGenerator.start
         end
       end
+
+      # We need custom treatment for `pem`
+      def module_name_for_tool(tool_name)
+        if tool_name == "pem"
+          return PEM
+        end
+        return tool_name.fastlane_class
```

Comment:
No, because this is actually specific to the module of `PEM`, and not the class name. Looking at the code base, moving this into `fastlane_class` would break the fastlane action of `PEM` 👍